x-pack/plugins/console_extensions/spec/overrides/ilm.put_lifecycle.json

```diff
@@ -0,0 +1,54 @@
+{
+  "ilm.put_lifecycle": {
+    "data_autocomplete_rules": {
+      "phases": {
+        "warm": {
+          "after": "",
+          "actions": {
+            "allocate": {
+              "number_of_replicas": "",
```

Comment:
it typically used a number here (say 1) because console will immediately populate this as an example when people select `number_of_replicas`. This makes it clear that a number is expected.

Reply:
done